.bingo/Variables.mk

```diff
@@ -1,4 +1,4 @@
-# Auto generated binary variables helper managed by https://github.com/bwplotka/bingo v0.6. DO NOT EDIT.
```

Comment:
This project still uses bingo 0.6, I think you accidentally used the wrong bingo version.